cmake: `.rodata' can not be used when making a shared object

```
cmake .. && make
...
[100%] Building C object pgsql/CMakeFiles/pointcloud.dir/pc_pgsql.c.o
Linking C shared module pointcloud.so
/usr/bin/ld: ../lib/libpc.a(pc_bytes.c.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
../lib/libpc.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
```

--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -33,6 +33,7 @@ set_target_properties (libpc-static
     OUTPUT_NAME "pc"
     PREFIX "lib"
     CLEAN_DIRECT_OUTPUT 1
+    COMPILE_FLAGS -fPIC
   )
 
 target_link_libraries (libpc-static xml2)

